src/app/components/contentTile/contentTile.component.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import './contentTile.component.scss';
+import { ContentItem, ContentLabel } from '../../modules/content/content.state';
+
+export interface ContentTileProps {
+    item: ContentItem;
+    label: ContentLabel;
+    specifiedLength: number;
+}
+
+export function ContentTile({ item, label, specifiedLength }: ContentTileProps) {
+    const summaryLength = item.summary.length;
+    const usedLength = specifiedLength > summaryLength ? summaryLength-3 : specifiedLength;
+
+    return (
+        <section
+            className="content-tile"
+            onClick={() => window.open(item.url, '_blank')}
+        >
+            <div className="for-small-screens">
```

Comment:
you should have one class for this template and differ styling by media queries

Reply:
to solve in #52